pkg/dir.props

```diff
@@ -28,7 +28,7 @@
   <PropertyGroup>
     <PackageOutputPath>$(PackagesOutDir)</PackageOutputPath>
     <SymbolPackageOutputPath>$(SymbolPackagesOutDir)</SymbolPackageOutputPath>
-    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\1.0.2\runtime.json</RuntimeIdGraphDefinitionFile>
+    <RuntimeIdGraphDefinitionFile>$(ProjectDir)packages\Microsoft.NETCore.Platforms\DEPENDENCY 'Microsoft.NETCore.Platforms' NOT FOUND\runtime.json</RuntimeIdGraphDefinitionFile>
```

Comment:
(To link them together: https://github.com/dotnet/core-setup/pull/787#discussion_r92452114)